src/menu.scss

```diff
@@ -5,165 +5,294 @@
+      }
+    }
+
+    .#{$block}__addon-before {
+      width: 2.75rem; // to account for item right padding of 16px or 1rem so 2.75rem-1rem = 1.75rem
+      height: 1.25rem; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem
+      margin: 0 0 0 -1rem;
+    }
 
-    text-transform: uppercase;
+    .#{$block}__addon-after {
+      width: 2.75rem; // to account for item right padding of 16px or 1rem so 2.75rem-1rem = 1.75rem
+      height: 1.25rem; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem
+      margin: 0 -1rem 0 0;
+    }
 
-    @include fd-var-color("color", $fd-menu-title-color, --fd-menu-title-color);
-    @include fd-var-color("background-color", $fd-menu-title-background-color, --fd-menu-title-background-color);
+    @include fd-rtl() {
+      .#{$block}__addon-before {
+        margin: 0 -1rem 0 0; // since before and after icons switch places, their margins also switch
+      }
+
+      .#{$block}__addon-after {
+        margin: 0 0 0 -1rem; // since before and after icons switch places, their margins also switch
+      }
+    }
+  }
+
+  &__separator {
+    height: 0.0625rem;
+    width: 100%;
+    background-color: $fd-menu-separator-color;
   }
 
   &__item {
     @include fd-reset();
-    @include fd-ellipsis();
 
-    display: block;
-    transition: all $fd-animation-speed ease-in;
+    display: flex;
+    align-items: center;
 
     @include action-cursor();
 
-    padding: $fd-menu-item-padding-y $fd-menu-item-padding-x;
+    max-width: 20rem;
+    background-color: $fd-menu-item-background-color;
+    // tablet cozy - will be default
+    padding: 0.75rem 1rem 0.75rem 0.75rem; // line-height=20px so 2.75rem or 44px-20px=24px/2= 12px or 0.75rem each
+    height: 2.75rem;
 
-    @include fd-var-color("background-color", $fd-menu-item-background-color, --fd-menu-item-background-color);
-    @include fd-var-color("color", $fd-menu-item-color, --fd-menu-item-color);
-    @include fd-weight("normal");
-
-    text-decoration: none;
+    @include fd-hover() {
+      .#{$block}__title {
+        color: $fd-menu-item-color;
+      }
 
-    > .fd-checkbox {
-      margin: 0 10px 0 0;
+      background-color: $fd-menu-item-background-color--hover;
     }
 
-    @include fd-hover() {
-      @include fd-var-color("color", $fd-menu-item-color--hover, --fd-color-text-1);
-      @include fd-var-color("background-color", $fd-menu-item-background-color--hover, --fd-color-background-hover);
+    @include fd-active() {
+      .#{$block}__title {
+        color: $fd-menu-item-color--active;
+      }
+
+      background-color: $fd-menu-item-background-color--active;
+
+      .#{$block}__addon-before,
+      .#{$block}__addon-after {
+        color: $fd-menu-primary-icon-color-active-selected;
+      }
+
+      .#{$block}__addon-after {
+        // border-bottom-color: var(--sapList_BorderColor); // TODO change this when fade() evaluates to a value
```

Comment:
you can remove this comment

Reply:
done. have pushed in the other changes as well, please check.